tst/memfs-dotnet/Program.cs

```diff
@@ -255,6 +255,7 @@ public override Int32 Init(Object Host0)
             Host.NamedStreams = true;
             Host.PostCleanupWhenModifiedOnly = true;
             Host.PassQueryDirectoryFileName = true;
+            Host.FlushAndPurgeOnCleanup = true;
```

Comment:
I would ask that we remove this line, because it changes the default behavior of WinFsp .NET file systems. Thanks to your work, It is now easy to have a file system set this property when it needs it.

Reply:
Sure. It was a test that I didn't remove ...